src/uct/api/uct.h

```diff
@@ -1078,6 +1091,25 @@ struct uct_cm_attr {
 };
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief UCT listener attributes, capabilities and limitations.
+ */
+struct uct_listener_attr {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_listener_attr_field. Fields not specified by this mask
+     * will be ignored.
+     */
+    uint64_t    field_mask;
+
+    /**
+     * Port number on which this listener is listening. (in host byte order)
+     */
+    int         listen_port;
```

Comment:
maybe return a sockaddr which includes both ip address and port? it's more general
by rdma_get_local_addr()